openapi.yml

```diff
@@ -4,7 +4,7 @@ info:
   title: Kytos Circuit Provisioning
   description: Kytos NApp for provisioning of SDN circuits.
 servers:
-  - url: http://localhost:8181/api/kytos/mef_eline
+  - url: http://0.0.0.0:8181/api/kytos/mef_eline
```

Comment:
openapi-core provide a way to keep the URL host agnostic: https://github.com/p1c2u/openapi-core/issues/218